packages/sanity/src/core/versions/util/const.ts

```diff
@@ -0,0 +1,38 @@
+/*  TEMPORARY  DUMMY DATA */
+
+import {type IconSymbol} from '@sanity/icons'
+import {type ButtonTone} from '@sanity/ui'
+
+import {type Version} from '../types'
+
+export const RANDOM_TONES: ButtonTone[] = ['default', 'caution', 'critical', 'positive', 'primary']
+export const RANDOM_SYMBOLS: IconSymbol[] = [
+  'archive',
+  'edit',
+  'eye-open',
+  'heart',
+  'info-filled',
+  'circle',
+  'search',
+  'sun',
+  'star',
+  'trash',
+  'user',
+]
+
+export const LATEST: Version = {
+  name: 'draft',
```

Comment:
Should this be `drafts` ? 
```suggestion
  name: 'drafts',
```

The id generated for the draft bundles is `drafts. + id`